assets/style/works/works.scss

```
@@ -5,3 +5,333 @@
+        }
+      }
+      .main__wrapper-common-picture {
+        margin-right: 47px;
+        @media (max-width: $laptop-l) {
+          width: 30%;
+        }
+        @media (max-width: $phone-l) {
+          display: none;
+        }
+      }
+      .main__wrapper-common-text-paragrph {
+        @include default-p($text-blue-color);
+        margin-top: 20px;
+      }
+    }
+    .main__wrapper-minerals-title {
+      margin-bottom: 20px;
+      margin-top: 40px;
+      font-weight: 700;
+      font-size: 32px;
+      line-height: 28px;
+      color: $text-black-color;
+      @media (max-width: $pc-m) {
+        margin-left: 20px;
+        margin-right: 20px;
+      }
+      @media (max-width: $phone-l) {
+        margin-top: 20px;
+      }
+    }
+    .main__wrapper-common-text-paragrph {
+      @include default-p($text-blue-color);
+      margin-bottom: 20px;
+      @media (max-width: $pc-m) {
+        margin-left: 20px;
+        margin-right: 20px;
+      }
+    }
+  }
+  .main__wrapper-common-best {
+    max-width: 1420px;
+    min-height: 98px;
+    display: flex;
+    @media (max-width: $pc-m) {
+      margin-left: 20px;
+      margin-right: 20px;
+    }
+    @media (max-width: $phone-m) {
+      flex-wrap: wrap;
+    }
+    .main__wrapper-common-best-item {
+      display: flex;
+      justify-content: start;
+      align-items: start;
+      margin-bottom: 50px;
+      @media (max-width: $phone-l) {
+        margin-bottom: 10px;
+      }
+
+      .main__wrapper-common-best-item-picture {
+        margin-right: 15px;
+      }
+      @media (max-width: $phone-l) {
+        flex-wrap: wrap;
+      }
+
+      .main__wrapper-common-best-item-text {
+        display: flex;
+        flex-direction: column;
+      }
+    }
+  }
+  .common-text-heading {
+    @include default-h($text-dark-color);
+    @media (max-width: $tablet-l) {
+      font-size: 35px;
+      line-height: 32px;
+    }
+  }
+  .title-best {
+    max-width: 181;
+    min-height: 38px;
+    font-family: $font;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 25px;
+    align-items: center;
+    color: $btn-link-dark-color;
+  }
+  .text-best {
+    font-family: $font;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 26px;
+    color: $text-blue-color;
+    max-width: 306px;
+    width: 100%;
+    min-height: 57px;
+  }
+  .main__wrapper-certificates {
+    margin-bottom: 46px;
+    .main__wrapper-certificates-picture {
+      @media (max-width: $pc-m) {
+        display: flex;
+        margin: 0 auto;
+        width: 70%;
+      }
+    }
+    .main__wrapper-certificates-common {
+      max-width: 513px;
+      min-height: 91px;
+      margin: 0 auto;
+      text-align: center;
+      margin-top: 100px;
+      margin-bottom: 20px;
+      @media (max-width: $phone-l) {
+        margin-top: 30px;
+        min-height: 41px;
+        margin-bottom: 5px;
+      }
+    }
+  }
+  .main__wrapper-common-why {
+    min-height: 432px;
+    display: flex;
```

У тебя же картинка прижата к левому краю, тебе следует прописать:
```suggestion
    display: flex;
    justify-content: space-between;
```